src/components/grapher/GrapherTool.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <g class="grapher-tool--tool-dots-container">
+    <circle
+      v-for="dot in grapherToolDots"
+      :key="`${dot.x}-${dot.y}-tool--dot`"
+      class="grapher-tool--dot"
+      :cx="dot.x"
+      :cy="dot.y"
+      r="0.7"
+      data-test="grapher-tool--dot"
+    />
+    <g v-if="showDotLabels">
+      <text
+        v-for="(dot, index) in grapherToolDots"
+        :key="`${dot.x}-${dot.y}-tool--dottext`"
+        class="grapher-tool--dottext"
+        :x="dot.x"
+        :y="dot.y - 1"
+        data-test="grapher-tool--dottext"
+      >
+        {{ dotLabels[index] }}
+      </text>
+    </g>
+    <polyline
+      v-if="showSelectionLasso"
+      class="grapher-tool--selection-lasso"
+      :points="selectionLasso"
+      fill="none"
+      data-test="grapher-tool--selection-lasso"
+    />
+  </g>
+</template>
+
+<script lang="ts">
+import Vue from "vue";
+import StuntSheetDot from "@/models/StuntSheetDot";
+
+/**
+ * Renders the field, the dots of the current stunt sheet, and pending dots
+ * generated from the tool in use
+ */
+export default Vue.extend({
+  name: "GrapherTool",
+  computed: {
+    grapherToolDots(): StuntSheetDot[] {
+      return this.$store.state.grapherToolDots;
+    },
+    showDotLabels(): boolean {
+      return this.$store.state.showDotLabels;
+    },
+    dotLabels(): string[] {
+      const dotLabels = this.$store.getters.getDotLabels;
+      const dots: StuntSheetDot[] = this.$store.state.grapherToolDots;
+      return dots.map((dot, index) => {
+        return dotLabels !== null &&
+          dot.dotLabelIndex !== null &&
+          dot.dotLabelIndex < dotLabels.length
+          ? dotLabels[dot.dotLabelIndex]
+          : index.toString();
+      });
+    },
+    showSelectionLasso(): boolean {
+      return this.$store.state.showSelectionLasso;
+    },
+    selectionLasso(): number[] {
+      return [].concat(...this.$store.state.selectionLasso);
+    },
```

Comment:
nit: Could you have this return a string? It would make it easier to make the connection to what `points` is supposed to look like https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/points. Something like
```suggestion
    selectionLasso(): string {
      const lassoPointStrings: string[] = this.$store.state.selectionLasso.map((point) => point.join(","))
      return lassoPointStrings.join(" ")
    },
```

Reply:
Yep!